x/stakeibc/migrations/v3/convert.go

```diff
@@ -0,0 +1,86 @@
+package v3
+
+import (
+	sdkmath "cosmossdk.io/math"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	oldstakeibctypes "github.com/Stride-Labs/stride/v13/x/stakeibc/migrations/v3/types"
+	newstakeibctypes "github.com/Stride-Labs/stride/v13/x/stakeibc/types"
+)
+
+var (
+	// The threshold, denominated in percentage of TVL, of when a slash query should
+	// be submitted (1 => 1%)
+	ValidatorSlashQueryThreshold uint64 = 1
+	// The exchange rate here does not matter since it will be updated after the slash query
+	// Setting it to this value makes it easier to verify that we've submitted the query
+	DefaultExchangeRate = sdk.MustNewDecFromStr("1.000000000000001")
+)
+
+// Converts an old validator data type to the new schema
+// Changes are as follows:
+//   - Added SlashQueryProgressTracker field
+//   - Added SlashQueryCheckpoint field
+//   - Added DelegationsInProgress field
+//   - Added SlashQueryInProgress field
+//   - InternalExchangeRate is now a decimal named SharesToTokensRate
+//   - DelegationAmt renamed to Delegation
+func convertToNewValidator(oldValidator oldstakeibctypes.Validator, totalDelegations sdkmath.Int) newstakeibctypes.Validator {
```

Comment:
Looks good once the questions re `DefaultExchangeRate` are addressed.
Confirming that you haven't missed any proto fields on `Validator`.